UPGRADING.md

```diff
@@ -51,4 +51,13 @@ After:
 - The `name` field, getter and builder method have been renamed to `repository`.
 
 ### CloneSnapshotRequest timeout removal
-- The `timeout` field, getter and builder method have been removed from `CloneSnapshotRequest` as it is not supported by OpenSearch.
\ No newline at end of file
+- The `timeout` field, getter and builder method have been removed from `CloneSnapshotRequest` as it is not supported by OpenSearch.
+
+### indices.Alias
+- The `indexRouting`, `routing` and `searchRouting` properties now accept a `List<String>` instead of a `String`.
```

Comment:
For later, this could be patched to accept `String` as well, right?

Reply:
Why is that? I believe it does not match the server model (`AliasActions`) where this properties are `String`s

Reply:
Have undone this change / tweaked the spec. Was a case of a re-used schema between the query string params which accept a comma-separated list and the body fields which only accept/return strings.